homeassistant/components/ps4/media_player.py

```diff
@@ -328,8 +328,7 @@ def media_content_id(self):
     @property
     def media_content_type(self):
         """Content type of current playing media."""
-        # No MEDIA_TYPE_GAME attr as of 0.90.
-        return MEDIA_TYPE_MUSIC
+        return MEDIA_TYPE_GAME
```

Comment:
I think the PS4 has apps too?

Reply:
The PS4 does also have apps, as far as I can tell we only have game id number and name, no identification of what type of item the "application" is. ie app or game, so since it is a gaming console, the item is more than likely going to be a game (for now)